Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -802,7 +806,9 @@
   - TO CONSUME...CONSUME EVERYTHING...
   - TO GO TO DISNEYLAND
   - TO GO TO SYNDIELAND
+  - TO MERGE MY FUCKING PULL REQUEST
```

Comment:
:godo:

Reply:
That's an annoyingly good point. Got an alternative? :)

Reply:
TO SUMMON OUR LORD CARP-SIE

Reply:
NAR-SIE probably works better actually 🤔